tests/AnalyzeSpec.hs

```diff
@@ -1222,6 +1229,16 @@ spec = describe "analyze" $ do
           |]
     in expectPass code $ Valid $ bnot Abort'
 
+--   describe "big round" $
```

Comment:
do we want to do anything with this test? remove it? or mark it pending something?